betterdiscordctl

```diff
@@ -148,15 +115,15 @@ while :; do
     --modules=)
       die_non_empty '--modules'
       ;;
-    --bd-repo-branch)
-      if [[ ${2+x} ]]; then bd_repo_branch=$2; shift
-      else die_non_empty '--bd-repo-branch'; fi
+    -b|--betterdiscord)
+      if [[ ${2+x} ]]; then asar=$2; shift
```

Comment:
Flag names should mirror variable names. Either rename the variable `asar` to `betterdiscord`, rename `--betterdiscord` to `--asar`, or rename both to something more descriptive. If renaming, `--bd-asar-file` & `bd_asar_file` feels good for its semantic complexity. I don't know what a good short option name would be, but we could drop the short option. We should probably rename the option from `--betterdiscord` given the semantic change anyways (specifying overall directory → specifying asar file).

Reply:
I think `-a/--bd-asar` may be better since we're using `-r/--bd-repo`.